### PR TITLE
JCR-4979: Adjust SCM urls after git migration

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: jackrabbit-parent/pom.xml [hunk 1]
```diff
@@ -725,9 +725,4 @@
         https://svn.apache.org/repos/asf/jackrabbit/site/trunk/src/site/markdown/jackrabbit-team.md
   -->
 
-  <scm>
```
Comment (kwin) [inline jackrabbit-parent/pom.xml:728]: I don't think this is ever released separately therefore on scm element on root level should suffice (similar to what we have in Oak)

Comment (reschke) [reply jackrabbit-parent/pom.xml:728]: Likely.

Comment (reschke) [reply jackrabbit-parent/pom.xml:728]: The release of 1.60.0 will be done tomorrow.

Maybe we should release 1.62.0 right away to make sure that things are correct and we notice ASAP if not?

Comment (kwin) [reply jackrabbit-parent/pom.xml:728]: This is about JR2 not about JR Oak.

Comment (reschke) [reply jackrabbit-parent/pom.xml:728]: Right. Sorry. Im meant 2.21.21.

Comment (kwin) [reply jackrabbit-parent/pom.xml:728]: Yes, makes sense to try out with Git soon afterwards.

**Comment by kwin** [comment]
The readme still references svn...

**Comment by kwin** [comment]
@reschke Any concerns with merging this?

**Review by reschke — APPROVED** [review]
LGTM; if something is wrong we will notice that when we cut 2.21.22 (plan: tomorrow)

**Pull request merged**